silveroak-opentitan/.gitignore

```diff
@@ -0,0 +1,10 @@
+Makefile.coq
```

Comment:
Adding this `.gitignore` seems like a good move, but it seems to me like it's not logically part of this change. Unless the change creates a bunch more generated files?

Reply:
I copied one of the existing .gitignore contents, to try and hopefully not miss additional files (alternatively, perhaps its too aggressive!)